examples/platforms/efr32mg12/README.md

```diff
@@ -1,6 +1,6 @@
-# OpenThread on EFR32 Example
+# OpenThread on EFR32MG12 Example
 
-This directory contains example platform drivers for the [Silicon Labs EFR32][efr32mg]
+This directory contains example platform drivers for the [Silicon Labs EFR32MG12][efr32mg12]
```

Comment:
Should we add the link for 'efr32mg12'?

Reply:
The link for EFR21MG12 is already added in the lines below.

Reply:
The link is already present in the line below.